### ISSUE TITLE
DBeaver: Final touch on the Generate UUID function

### ISSUE THREAD

**Issue by pdanie** [dbeaver/dbeaver#4901]
DBeaver - **Version 5.3.2 Beta CE**
DBeaver driver: **MS SQL Server  / Microsoft Driver**, **PostgreSQL Driver**, etc.
Operating System: **Windows 7 / Windows 8.1 / Windows 10**
Database Server: **Microsoft SQL Express 2014, 2016, 2017**

Minor improvement to the **Generate UUID** function from the #4818 issue, might be:

**o** Add the keyboard shortcut to the **Edit Cell** function popup **Form** dialog.

> If I right click on the grid cell and choose **Edit** -> **Edit Cell** _(Shift+Enter)_ that **Form** dialog does not react on the **Ctrl+Shift+U** keyboard shortcut.
> 
> ![2019-01-02 21-29-16](https://user-images.githubusercontent.com/31191374/50611214-e982d800-0ed5-11e9-9dc5-8185c87fcba5.png)
> 
> **PS**. Strange that the dialog is named just **Form**. Should it not be named **Edit cell** or something?
> 

**o** Add a menue choice in the data grid context menu for **Generate UUID** _(Ctrl+Shift+U)_.

> ![2019-01-02 22-01-19](https://user-images.githubusercontent.com/31191374/50612444-58623000-0eda-11e9-89c6-24e43041db4d.png)

**Comment by chapmanjacobd** [comment]
Yeah it seems like the Generate UUID option only generates the same UUID for all rows as well in v21.1.0

**Comment by Shizukaax** [comment]
> Yeah it seems like the Generate UUID option only generates the same UUID for all rows as well in v21.1.0

This issue still persists in v22.2.1, will there be any fixes for this?
